apps/google-analytics-4/lambda/package.json

```diff
@@ -34,14 +34,15 @@
     "mocha": "^10.2.0",
     "nodemon": "^2.0.20",
     "rimraf": "^3.0.2",
-    "serverless": "^3.26.0",
-    "serverless-domain-manager": "^6.4.3",
-    "serverless-offline": "^12.0.4",
+    "serverless": "2.72.3",
```

Comment:
😭

Reply:
would love to see some notes/comments left as to why we needed to revert to an older version. this concerns me from a tech debt/vulnerability standpoint and I want to make sure it's clear to future-us why we did this.

Reply:
my main concern about this by the way is dependabot issues this has a cascading affect on. there's several `critical` dep VULNs that cannot be resolved due to pinned serverless in other apps lamba's already so this will surely have the same problem very soon if not immediately :/